File: lab4/src/main/java/ki306/yarema/lab4/models/CommanderGasMask.java
package ki306.yarema.lab4.models;

public class CommanderGasMask extends GasMask implements CommanderEquipment {

    @Override
    public String getEquipmentType() {
        return getType();
    }

    @Override
    public void sendSosSignal() {
        log("Sending SOS signal...");
    }
}

File: lab9Python/commander_gasMask.py
from gasMask import GasMask

class CommanderGasMask(GasMask):
    def __init__(self, mask_type, filter_type):
        super().__init__(mask_type, filter_type)

    def get_equipment_type(self):
        return self.get_type()

    def send_sos_signal(self):
        print("Sending SOS signal...")